Persistence/HaSpManContext.cs

```diff
@@ -22,9 +22,16 @@ public HaSpManContext(DbContextOptions<HaSpManContext> options)
     public DbSet<BankAccount> BankAccounts { get; set; } = null!;
     public DbSet<BankAccountsWithTotals> BankAccountsWithTotals { get; set; } = null!;
     public DbSet<Transaction> Transactions { get; set; } = null!;
+    public DbSet<FinancialYear> FinancialYears { get; set; } = null!;
 
     protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
-            => optionsBuilder.UseSqlServer(x => x.MigrationsHistoryTable("__EFMigrationsHistory", Schema.HaSpMan));
+    {
+        if (!optionsBuilder.IsConfigured)
```

Comment:
Why do we need this?

Reply:
For unit testing purposes, we can't have the provider already configured.